functions.php

```diff
@@ -1,4 +1,25 @@
 <?php
+
+/**
+ * Checks whether the WordPress REST API plugin is active.
+ *
+ * Adds an admin notice and switches to the default theme if the plugin is not active.
+ */
+function picard_check_rest_api_dependency() {
+	if ( ! is_plugin_active( 'json-rest-api/plugin.php' ) ) {
```

Comment:
@fklein-lu I do not think this conditional will work long term because it assumes the user has not changed the name of the plugin directory. Since Picard currently works with version one of WP-API, you could use:

```php
if ( defined( 'JSON_API_VERSION' ) )
```

However, for version two you would need to use:
```php
if ( defined( 'REST_API_VERSION' ) )
```